scipy/stats/tests/test_axis_nan_policy.py

```diff
@@ -125,15 +124,15 @@ def yeojohnson_llf(data, lmb, axis=0, _no_deco=False, **kwargs):
     (stats.brunnermunzel, tuple(), dict(distribution='normal'), 2, 2, False, None),
     (stats.mood, tuple(), {}, 2, 2, False, None),
     (stats.shapiro, tuple(), {}, 1, 2, False, None),
-    (stats.ks_1samp, (norm().cdf,), dict(), 1, 4, False,
+    (stats.ks_1samp, (special.ndtr,), dict(), 1, 4, False,
      lambda res: (*res, res.statistic_location, res.statistic_sign)),
-    (stats.ks_1samp, (norm().cdf,), dict(alternative='greater'), 1, 4, False,
+    (stats.ks_1samp, (special.ndtr,), dict(alternative='greater'), 1, 4, False,
      lambda res: (*res, res.statistic_location, res.statistic_sign)),
-    (stats.ks_1samp, (norm().cdf,), dict(alternative='less'), 1, 4, False,
+    (stats.ks_1samp, (special.ndtr,), dict(alternative='less'), 1, 4, False,
      lambda res: (*res, res.statistic_location, res.statistic_sign)),
     (stats.ks_2samp, tuple(), dict(), 2, 4, False,
      lambda res: (*res, res.statistic_location, res.statistic_sign)),
-    (stats.kstest, (norm().cdf,), dict(), 1, 4, False,
+    (stats.kstest, (special.ndtr,), dict(), 1, 4, False,
```

Comment:
Just confirming--here and elsewhere, this conversion isn't strictly needed--just slightly faster and avoids the typing suppression?

Reply:
Yeah, these tests are only running with NumPy, so `norm().cdf` still works, but I was switching a lot of other tests from using the old distributions to the using the special functions directly, so I went ahead and changed these, too. As you said, faster and avoids the mypy complaint.